ballerina/client.bal

```diff
@@ -32,6 +34,11 @@ public isolated client class Client {
     # + return - An `sql:Error` if the client creation fails
     public isolated function init(string url, string? user = (), string? password = (),
         Options? options = (), sql:ConnectionPool? connectionPool = ()) returns sql:Error? {
+        if strings:startsWith(url, "jdbc:sqlserver") &&
+                    (options is () || (options is Options && options.requestGeneratedKeys == ALL)) {
+            log:printWarn("Change the requestGeneratedKeys configuration as the MSSQL database does not support" + 
+                    "the retrieval of auto-generated keys with the batch execute function.");
```

Comment:
Shouldnt we return an error here? We know for MSSQL it will return vastly different error in this case. Since master branch is updated a minor version for Update4 we can add this change too.

Also review the message. This does not look professional.

Reply:
I tried to clean the history. After the push, this PR automatically closed. Couldn't re-open. Therefore, I have created a new PR: https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/769